### DISCUSSION TITLE
Does the engine usually take this long to load?

### DISCUSSION THREAD

**Discussion by Nekofinity26** [TrianglyRU/OrbinautFramework#25] [Q&A] [answered]
It's been around 4 minutes, and it doesn't seem to have compiled at all. (This is my first time compiling it btw) I was wondering if anybody was encountering the same issue. If so, is there a way to fix this?

![Screenshot (87)](https://github.com/user-attachments/assets/4bc46eb3-e0b0-4963-ada3-4b4e27f31616)

**Comment by TrianglyRU** [comment, marked as answer]
I see you have an the older IDE, `2023.2.1.75`. The current IDE is `2024.8.0.169`, so this might be the reason
